### DISCUSSION TITLE
Important discussion over at nvim-tree.lua

### DISCUSSION THREAD

**Discussion by cseickel** [nvim-neo-tree/neo-tree.nvim#549] [Internal Development]
I just want to let people know that [kyazdani42](https://github.com/kyazdani42) opened a discussion on the nvim-tree.lua repo about neo-tree succeeding nvim-tree. Since we don't have any log term maintainers on this project other than myself, I just want to post it here so that anyone interested enough to be subscribed to alerts on discussions here can join the conversation if they like:

https://github.com/kyazdani42/nvim-tree.lua/issues/1613

**Comment by nyngwang** [comment]
In short, I don't think merging is a good idea. But [the point 1. you just said](https://github.com/kyazdani42/nvim-tree.lua/issues/1613#issuecomment-1264560930) is a good idea:

> nvim-tree becomes a configuration of neo-tree

---

The following are my random replies to some random comments on that thread.

I am a simple user that migrated from nvim-tree(some issue annoyed me for months without a solution) to neo-tree, and have been using neo-tree since its start. Just let me put some simple comments on the points I'm interested in:

> Also we should think of new users. When they come to neovim, why is there multiple trees written in lua ? [link](https://github.com/kyazdani42/nvim-tree.lua/issues/1613#issue-1390786730)

I did have this confusion on every kind of plugin of neovim. This is not avoidable, and not the responsibility of plugin maintainers. Because most of the time users themselves didn't know what they were asking for. The solution for me is simple: I will choose a plugin that is actively maintained(ex. the current state of neo-tree), so I can join the discussion and, fortunately, one of the maintainers might agree with my point and make some changes.

> I get roped in when others submit PRs where the features are not quite complete in my eyes and I just add some polish to it.

I think you should keep doing it. Seriously, If a plugin has too many functionalities that are unrelated to my need(s) and are not easy to be disabled by configuration, I will uninstall it to avoid performance degration and inter-feature bugs and breaking changes. I prefer those features that are too opinionated to be done in a separate plugin.
